README.md

```diff
@@ -6,17 +6,20 @@ trackpy
 What is it?
 -----------
 
-**trackpy** is a Python package providing tools for particle tracking.
+**trackpy** is a Python package providing tools for 2D and 3D particle tracking.
```

Comment:
Yeah, I was thinking about that. "3D" needs to appear for… well, for SEO. But I should probably add "nD" to the list.